Source/gui/SNTAccessibleTextField.h

```diff
@@ -1,23 +0,0 @@
-/// Copyright 2016 Google Inc. All rights reserved.
-///
-/// Licensed under the Apache License, Version 2.0 (the "License");
-/// you may not use this file except in compliance with the License.
-/// You may obtain a copy of the License at
-///
-///    http://www.apache.org/licenses/LICENSE-2.0
-///
-///    Unless required by applicable law or agreed to in writing, software
-///    distributed under the License is distributed on an "AS IS" BASIS,
-///    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-///    See the License for the specific language governing permissions and
-///    limitations under the License.
-
-#import <Cocoa/Cocoa.h>
-
-/**
-   An NSTextField subclass that provides an accessiblity label equal to:
-   (self.toolTip + self.stringValue) where available. It also sets the
-   accessibilityRoleDescription to "label".
-*/
-@interface SNTAccessibleTextField : NSTextField
```

Comment:
Hmm, the current uses `SNTAccessibleTextField`. The .xib uses it as a custom class for each text field, along with adding a tooltip.

In what order are the fields traversed when accessibility is enabled with the new Swift UI? I assume they follow the V stack. If that is the case, we should add tooltips and accessibility labels for the values in the main popup. The details modal should be okay with out additional labels, since the field name will come right before the value in the accessibility tree.

Reply:
It reads them in the correct order; from the screenshot:

AppIcon, Image -> Santa, text -> This application has been banned, text -> Application, text -> Spotify, text -> Publisher, text -> Spotify - Developer ID Application: Spotify (2FNC3A47ZF), text -> User, text -> rah, text -> More details, button -> Copy details, button, Prevent future notifications for this application for a day, checkbox -> Open sync server, button -> Dismiss, button

Reply:
Awesome!